ideascube/monitoring/tests/test_views.py

```diff
@@ -325,6 +358,48 @@ def test_cant_create_inventoryspecimen_by_barcode_twice(staffapp):
                                             specimen=specimen).count() == 1
 
 
+def test_staff_can_create_inventoryspecimen_by_serial(staffapp):
```

Comment:
What about trying to create an inventory barcode by serial if it previously was created by serial? 😜